docs/workstation_release_management.rst

```diff
@@ -259,17 +180,52 @@ production (``yum.securedrop.org``) using the following procedure:
 16. Upon merge to master, ensure that changes deploy to
     ``yum.securedrop.org`` without issue.
 
-Signing binaries/packages
--------------------------
-
-Debian packages
-~~~~~~~~~~~~~~~
-
-The apt repository Release file will be signed, containing checksums of
-the debs.
-
-RPM packages
-~~~~~~~~~~~~
+Signing procedures
+==================
+
+.. _Sign the tag with the SecureDrop release key:
+
+Sign the tag with the SecureDrop release key
+--------------------------------------------
+
+1. If the tag does not already exist, create a new release tag: ``git tag -a VERSION``.
+2. Output the tag to a file: ``git cat-file tag VERSION > VERSION.tag``.
+3. Copy the tag file into your signing environment and then verify the tag commit hash.
+4. Sign the tag with the SecureDrop release key: ``gpg --armor --detach-sign VERSION.tag``.
+5. Append ASCII-armored signature to tag file (ensure there are no blank lines): ``cat VERSION.tag.sig >> VERSION.tag``.
+6.  Move tag file with signature appended back to the release environment.
+7. Delete old (unsigned) tag: ``git tag -d VERSION``.
+8. Create new (signed) tag: ``git mktag < VERSION.tag > .git/refs/tags/VERSION``.
+9. Verify the tag: ``git tag -v VERSION``.
+10. Push the tag to the shared remote: ``git push origin VERSION``.
+
+.. _Regenerate and sign the apt release file:
+
+Regenerate and sign the apt release file
+----------------------------------------
+
+1. From the ``release`` branch containing the new package, update the apt repository distribution files.
+```shell
+git clone https://github.com/freedomofpress/securedrop-debian-packages-lfs
+cd securedrop-debian-packages-lfs
+git checkout -b release
+./tools/publish`
+```
```

Comment:
The suggestion tool gets confused by the ` ``` `, but I wanted to suggest indenting the code blocks in this list (3 spaces, so that they become part of each item).